model-optimizer/extensions/middle/AddMeanScaleValues_test.py

```diff
@@ -44,7 +46,19 @@
 }
 
 
+def set_graph_attrs(graph, graph_ref):
+    for node in graph.get_data_nodes():
+        node['fw_tensor_debug_info'] = ['data_attributes', 0]
+    for graph_ref in graph_ref.get_data_nodes():
+        graph_ref['fw_tensor_debug_info'] = ['data_attributes', 0]
+
+
 class AddMeanScaleValuesTest(unittest.TestCase):
+    def check_graph_attrs(self, graph, graph_ref):
+        for node_name in graph.get_nodes_with_attributes():
+            if 'fw_tensor_debug_info' in graph.node[node_name]:
+                self.assertTrue(graph.node[node_name]['fw_tensor_debug_info'] == graph_ref.node[node_name]['fw_tensor_debug_info'])
```

Comment:
This function should check that `Parameter`s data nodes have the same value for `'fw_tensor_debug_info'` attribute and that the rest of data nodes does not have it.